python-avd/pyavd/_eos_designs/schema/schema_fragments/router_general.schema.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  router_general_settings:
```

Comment:
I don't see why we need this. You should just pick up the router id IPs as we normally do. The boolean is just changing how we output the config.

Reply:
Removed.